froide/foirequest/templates/foirequest/header/header.html

```diff
@@ -103,82 +92,104 @@ <h2 class="request-title">{{ object.title }}</h2>
             {# project #}
             <div class="mt-3 text-gray-500">
               {% blocktrans %}Project:{% endblocktrans %}
-              <a class="text-gray-500 fw-bold" href="{{ object.project.get_absolute_url }}">
-                {{ object.project.title }}
-              </a>
+              <a class="text-gray-500 fw-bold"
+                 href="{{ object.project.get_absolute_url }}">{{ object.project.title }}</a>
             </div>
           {% endif %}
         {% endif %}
 
         {# law if not meta #}
         {% if object.law and not object.law.meta %}
-        <div class="mt-3 text-gray-500">
-          {% blocktrans %}Legal basis:{% endblocktrans %}
-          <a class="text-gray-500 fw-bold" href="{{ object.law.get_absolute_url }}">{{ object.law.name }}</a>
-        </div>
+          <div class="mt-3 text-gray-500">
+            {% blocktrans %}Legal basis:{% endblocktrans %}
+            <a class="text-gray-500 fw-bold"
+               href="{{ object.law.get_absolute_url }}">{{ object.law.name }}</a>
+          </div>
         {% endif %}
 
         {# description #}
         <div class="mt-3">
           <div class="request-descr request-descr--collapsed">
-            {{ object.get_description|urlizetrunc:40|linebreaks }}
+
+            {% redact_request_description object request as redacted_description %}
+            {{ redacted_description|urlizetrunc:40|linebreaks }}
```

Comment:
`redact_description` is already 'urlized' in `mark_redacted`, so this would apply `urlizetrunc` twice.